src/main/java/net/masterthought/cucumber/ReportBuilder.java

```diff
@@ -131,7 +131,10 @@ private void copyStaticResources() {
     }
 
     private void createEmbeddingsDirectory() {
-        configuration.getEmbeddingDirectory().mkdirs();
+        final File embeddingDirectory = configuration.getEmbeddingDirectory();
+        if(!embeddingDirectory.exists() && !embeddingDirectory.mkdirs()){
```

Comment:
looks like it introduces more than just logger update - if so, split into two separate PRs

Reply:
ping

Reply:
@aravinder111 broke this out for @AndyGee into a separate PR: #718